### ISSUE TITLE
Something is wrong with public server listings

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#8029]
Noticed this myself when Eric says theres thousands of servers but only 50 or so public.  This user is confirming it as well:

![image](https://user-images.githubusercontent.com/3536496/41257506-f05f9f82-6d81-11e8-9f90-cdee65d71bff.png)

**Comment by MarjieAVolk** [comment]
We've compared rough counts between how many servers are public and how many are shown in the server browser, and they seem to be about right.  You do have to go in and disable some of the default filters in order to see the complete list, and it's not totally clear which filter options are more or less constrained at first glance, so that might be where the confusion is coming from.

I do know that some servers take a very long time to load into the server browser.  That seems to be what this person is describing, since they say the server is sometimes found.  Whether people see this for their server depends on where they are in the master server list, just based on random factors I believe, or maybe on how long the server has been running.  I recently increased the number of servers processed per frame from 1 to 5, so people looking for a specific server should see it load in 5x faster than it did before.  I believe this change is going out with 7.6 (see https://github.com/StrangeLoopGames/EcoIssues/issues/7836).

Other than that, there are 2 other issues described in this review.  One, they have some kind of connection error, which has no details.  Two, the report a bug feature isn't accepted in their browser (I presume this is a browser specific issue), because the url is too long.  If we didn't attach the client log, this wouldn't be an issue.  I would vote for removing it, since it is rarely useful and has caused several reports of issues like this.
